cli/rotate.go

```diff
@@ -28,6 +29,10 @@ func ConfigureRotateCommand(app *kingpin.Application) {
 		Short('t').
 		StringVar(&input.MfaToken)
 
+	cmd.Flag("mfa-serial-override", "Override the MFA Serial defined in AWS Profile").
```

Comment:
The "override" is redundant, as CLI flags always override other config. I think `mfa-serial` for the flag and something like `The identification number of the MFA device to use`